elements/lisk-transactions/src/transfer.ts

```diff
@@ -140,7 +152,13 @@ export const transfer = (inputs: TransferInputs): Partial<TransactionJSON> => {
 		transactionWithSenderInfo,
 	);
 
-	transferTransaction.sign(passphrase);
+	if (passphrase) {
+		transferTransaction.sign(networkIdentifier, passphrase);
```

Comment:
shouldn't this return here? as it says `// If senderPassphrase is passed in assume only one signature required`?
or maybe add a test where it covers when `passphrase` and `passphrases` are given?

Reply:
Changed to return on each case